pkg/validator/server.go

```diff
@@ -2,19 +2,27 @@
 package validator
 
 import (
+	"context"
+	"crypto/tls"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 
+	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
 	admissionv1 "k8s.io/api/admission/v1"
+	"k8s.io/client-go/kubernetes"
 
+	"github.com/openservicemesh/osm/pkg/certificate"
 	"github.com/openservicemesh/osm/pkg/webhook"
 )
 
 var (
 	defaultValidators = map[string]Validator{}
+
+	// ValidatingWebhookPath is the path for the validating webhook.
+	ValidatingWebhookPath = "/validator/validate"
```

Comment:
What are the paths going to be for validating the different API types?

Reply:
right now they can all be added to the same path, and the server maintains a map of GVK to validator func

Reply:
Maybe just have the path set to `/validate`?

Reply:
done!